.env.example

```
@@ -3,6 +3,7 @@ NODE_ENV=production/development
 # Discord Bot
 DISCORD_BOT_TOKEN_PROD=your-bot-token-here
 DISCORD_BOT_TOKEN_DEV=your-bot-token-dev-here
+BOT_BASE_URL=http://localhost:3000 # e.g., https://your-bot-website.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BOT_BASE_URL key should go before the DISCORD_BOT_TOKEN_DEV key

(UnorderedKey)